catalog/services_state.go

```diff
@@ -320,12 +331,28 @@ func (state *ServicesState) AddServiceEntry(newSvc service.Service) {
 	}
 }
 
+// Returns a service for a given ID if it happens to exist on the current
```

Comment:
I know the old comments aren't in GoDoc style, but let's make the new ones correct

Reply:
Oopsie :D Fixed, thanks!